### PR TITLE
Add docs for adding tasks to data downloader

### PR DESCRIPTION
**Pull request by jeswan**
This PR adds documentation for:

1. Adding a task to the data downloader (now required when adding a task)
2. @ the creator of a task when opening a PR adding a task

### CODE REVIEW

File: guides/tasks/adding_tasks.md [hunk 1]
```diff
@@ -162,9 +162,46 @@ def get_evaluation_scheme_for_task(task) -> BaseEvaluationScheme:
...
 ```
 
-## 4. Update the supported tasks documentation
+## 4. Add a data downloader
+If your task is publicly available, add the task to the data downloader in [`jiant/scripts/download_data/runscript.py`](../../jiant/scripts/download_data/runscript.py). There are two flavors of downloading tasks:
+
+1) If the task is supported by Hugging Face's [Datasets](https://huggingface.co/nlp/viewer/):
+    - Add the task to `OTHER_HF_DATASETS_TASKS` in [`jiant/scripts/download_data/constants.py`](../../jiant/scripts/download_data/constants.py). Also add the task to `HF_DATASETS_CONVERSION_DICT` in [`jiant/scripts/download_data/dl_datasets/hf_datasets_tasks.py`](../../jiant/scripts/download_data/dl_datasets/hf_datasets_tasks.py). `HF_DATASETS_CONVERSION_DICT` is used to map field changes in Dataset's to the original dataset.
+
+2) If the task is not supported by Hugging Face's Datasets, and the dataset is publicly available for download:
```
Comment (zphang) [inline guides/tasks/adding_tasks.md:171]: (i.e. downloadable directly via `wget`, and not behind an authentication wall such as Google Drive)

File: guides/tasks/adding_tasks.md [hunk 2]
```diff
@@ -162,9 +162,46 @@ def get_evaluation_scheme_for_task(task) -> BaseEvaluationScheme:
...
+    },
+    path=task_config_path,
+)
+```
+
+## 5. Update the supported tasks documentation
 Add your task to [`guides/tasks/supported_tasks.md`](../../guides/tasks/supported_tasks.md).
 
+
+## 6. Tag the creator in the pull request
```
Comment (zphang) [inline guides/tasks/adding_tasks.md:201]: Recommended:

File: guides/tasks/adding_tasks.md [hunk 3]
```diff
@@ -162,9 +162,46 @@ def get_evaluation_scheme_for_task(task) -> BaseEvaluationScheme:
...
+Please @ the creator of the task in your PR to let them know it is part of `jiant`! This also gives them a chance to review the task.
+
+
 ## Congratulations!
```
Comment (zphang) [inline guides/tasks/adding_tasks.md:205]: Mind tweaking the wording in this section to say adding a "new task" where Senteval is an example?

**Review by zphang — CHANGES_REQUESTED** [review]

File: guides/tasks/adding_tasks.md (hunk 1 above)
Comment (jeswan) [inline guides/tasks/adding_tasks.md:171]:
```suggestion
2) If the task is not supported by Hugging Face's Datasets, and the dataset is publicly available for download (i.e. downloadable directly via wget, and not behind an authentication wall such as Google Drive):
```

File: guides/tasks/adding_tasks.md (hunk 2 above)
Comment (jeswan) [inline guides/tasks/adding_tasks.md:201]:
```suggestion
## 6. (Recommended) Tag the creator in the pull request
```

**Review by zphang — APPROVED** [review]

**Pull request merged**